docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Categorical/OneHotEncoding.cs

```diff
@@ -1,6 +1,5 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Linq;
 using Microsoft.ML.Data;
 using static Microsoft.ML.Transforms.OneHotEncodingEstimator;
```

Comment:
what is this for? can we remove it?

Reply:
this is for the `OutputKind.Key` parameter that we use in the example below 

---
In reply to: [272292647](https://github.com/dotnet/machinelearning/pull/3179#discussion_r272292647) [](ancestors = 272292647)